src/materials/CrazyKCPlantFits.C

```diff
@@ -15,6 +15,7 @@ InputParameters
 CrazyKCPlantFits::validParams()
 {
   InputParameters params = ADMaterial::validParams();
+  params.addClassDescription("Material properties corresponding to KC plant reference material.");
```

Comment:
what about something more generic here, like "...corresponding to a demonstration reference material"? or maybe just "demonstration material"?